### PR TITLE
fix: margin on select component

### PR DESCRIPTION
**Pull request by birobirobiro**
Before:

image

After:

image

### CODE REVIEW

File: .env.template
Comment (brunocroh) [inline .env.template:1]: This file cannot be deleted.

File: pnpm-lock.yaml
Comment (brunocroh) [inline pnpm-lock.yaml:1]: Reset this change on pnpm-lock.yaml

File: apps/web/components/video-player.tsx
Comment (brunocroh) [inline apps/web/components/video-player.tsx:1]: We updated the prettier and add more features to this component, so could you please solve this conflicts to me be able to merge it?

**Review by brunocroh — CHANGES_REQUESTED** [review]
Firstly, I really appreciate your contribution, so thank you for it.

I just need you to reset the changes to `.env.template` and `pnpm-lock.yaml` so I can merge them.

File: apps/web/components/video-player.tsx
Comment (birobirobiro) [reply apps/web/components/video-player.tsx:1]: Sure! I'll resolve the conflicts and push the updates tonight.

**Pull request closed without merging**